coalib/settings/Setting.py

```diff
@@ -163,7 +163,11 @@ def __path__(self, origin=None, glob_escape_origin=False):
         :raises ValueError:        If no origin is specified in the setting
                                    nor the given origin parameter.
         """
-        strrep = str(self).strip()
+        if hasattr(self, 'value'):
+            strrep = self.value.strip()
```

Comment:
I think it'd be good to have a comment with a link to the issue here otherwise someone will come and remove this, figure out it doesn't work, file another issue, ...